doc/audit/index.rst

```diff
@@ -0,0 +1,24 @@
+Audit
+=====
+
+The high-integrity guarantees of CCF are enforced by a unique :term:`Merkle Tree` which records all transactions that mutate the key-value store. The root of the Merkle Tree is regularly signed by the primary node and the signature is recorded in the ``ccf.signatures`` key-value map. Like any other transaction, the signature transaction is also recorded in the ledger, which allows for offline auditability of the service.
```

Comment:
```suggestion
The high-integrity guarantees of CCF are enforced by a single :term:`Merkle Tree` which records all transactions that mutate the key-value store. The root of the Merkle Tree is regularly signed by the primary node and the signature is recorded in the ``ccf.signatures`` key-value map. Like any other transaction, the signature transaction is also recorded in the ledger, which allows for offline auditability of the service.
```

Although our implementation is probably not widely used, it's fairly typical, I think you meant to emphasise that there's only one instance of the tree?